requirements.txt

```diff
@@ -1,34 +1,37 @@
-fastapi==0.104.1
-fastapi-users[all]==10.4.1
+fastapi==0.109.2
+fastapi-users[all]==12.1.3
+# Can be updated only after migration to SQLAlchemy > 2.0
 fastapi-users-db-sqlalchemy==4.0.5
 uvicorn==0.27.1
-asyncpg==0.28.0
-psycopg2-binary==2.9.6
-pydantic==2.4.2
-pydantic-settings==2.0.3
-SQLAlchemy==1.4.46
-aiohttp==3.8.6
+asyncpg==0.29.0
+psycopg2-binary==2.9.9
+pydantic==2.6.1
+pydantic-settings==2.1.0
+SQLAlchemy==1.4.51
+aiohttp==3.9.3
 aiohttp-retry==2.8.3
-aiofiles==23.1.0
+aiofiles==23.2.1
 aioredis==2.0.1
-alembic==1.11.1
-pytest==7.3.1
+alembic==1.13.1
+alembic-postgresql-enum==1.1.1
+pytest==8.0.0
 pytest-cov==4.1.0
 jmespath==1.0.1
-PyYAML==6.0
-lxml==4.9.2
-syncer==1.3.0
-dramatiq[rabbitmq, watch]==1.14.2
-python-multipart==0.0.6
-plumbum==1.8.1
+PyYAML==6.0.1
+lxml==5.1.0
+syncer==2.0.3
+dramatiq[rabbitmq, watch]==1.16.0
+python-multipart>=0.0.7
+plumbum==1.8.2
 tap.py==3.1
-jinja2==3.1.2
+jinja2==3.1.3
 pgpy==0.6.0
-markdown==3.4.3
-httpx-oauth==0.11.2
-feedgen==0.9.0
+markdown==3.5.2
+httpx-oauth==0.13.2
+feedgen==1.0.0
 websockets==12.0
-sentry-sdk[fastapi]==1.28.1  # requires FastAPI>=0.79.0
+email-validator>=2.0
+sentry-sdk[fastapi]==1.40.4  # requires FastAPI>=0.79.0
 git+https://github.com/AlmaLinux/immudb-wrapper.git@0.1.2#egg=immudb_wrapper
 git+https://github.com/AlmaLinux/errata2osv.git@0.0.3#egg=errata2osv
-git+https://github.com/AlmaLinux/albs-github-integration.git@0.1.0#egg=albs_github
+git+https://github.com/AlmaLinux/albs-github-integration.git@0.1.1#egg=albs_github
```

Comment:
Did you release a new version of albs-github-integration ? I can't see 0.1.1 version of that library

Reply:
https://github.com/AlmaLinux/albs-github-integration/releases/tag/0.1.1 yes, it has updated dependencies